### PR TITLE
[Tachyon-449] Simplify constructor parameters of StorageTier

### PR DESCRIPTION
**Pull request by shimingfei**
Since all information can be gotten from TachyonConf

### CODE REVIEW

File: core/src/main/java/tachyon/worker/WorkerStorage.java [hunk 1]
```diff
@@ -785,42 +785,8 @@ public void initializeStorageTiers() throws IOException {
...
-      String[] dirCapacityStrings = tierDirsQuota.split(",");
-      // The storage directory quota for each storage directory
-      long[] dirCapacities = new long[dirPaths.length];
-      for (int i = 0; i < dirPaths.length; i ++) {
-        // If not all dir quotas are specified, the last one will be used for the quota for
-        // remaining dirs.
-        int j = Math.min(i, dirCapacityStrings.length - 1);
-        dirCapacities[i] = CommonUtils.parseSpaceSize(dirCapacityStrings[j].trim());
-      }
       StorageTier curTier =
```
Comment (calvinjia) [inline core/src/main/java/tachyon/worker/WorkerStorage.java]: Can this fit on one line now?

File: core/src/test/java/tachyon/worker/tiered/StorageTierTest.java [hunk 2]
```diff
@@ -64,29 +64,8 @@ public final void before() throws IOException, InvalidPathException {
...
-      long[] dirCapacities = new long[dirPaths.length];
-      for (int i = 0; i < dirPaths.length; i ++) {
-        // The storage directory quota for each storage directory
-        int j = Math.min(i, strDirCapacities.length - 1);
-        dirCapacities[i] = CommonUtils.parseSpaceSize(strDirCapacities[j].trim());
-      }
       StorageTier curTier =
-          new StorageTier(level, storageLevelAlias, dirPaths, dirCapacities, "/data", "/user",
-              nextTier, null, tachyonConf, new WorkerSource(null));
+          new StorageTier(level, tachyonConf, nextTier, new WorkerSource(null));
```
Comment (calvinjia) [inline core/src/test/java/tachyon/worker/tiered/StorageTierTest.java]: Can this be one line now?

**Comment by shimingfei** [comment]
@calvinjia  I have modified the code according to your comments, Thanks!

**Comment by calvinjia** [comment]
+1 LGTM

**Comment by apc999** [comment]
LGTM

**Pull request merged**